### PR TITLE
add localization to database

### PR DESCRIPTION
**Pull request by UAKovalRoman**

### CODE REVIEW

**Review by da-real-ivanpan — APPROVED** [review]

File: CharlieBackend.Api/SwaggerExamples/AccountsController/SignInResponse.cs [hunk 1]
```diff
@@ -13,6 +13,8 @@ internal class SignInResponse : IExamplesProvider<SignInResponse>
 
         public Dictionary<string, string> role_list { get; set; }
 
+        public string localization { get; set; }
```
Comment (DmitriyFesak) [inline CharlieBackend.Api/SwaggerExamples/AccountsController/SignInResponse.cs:16]: Please name the property with the capital letter

File: CharlieBackend.Business/Services/CurrentUserService.cs [hunk 2]
```diff
@@ -87,6 +88,28 @@ public string Email
...
+            {
+                if (_localization is null)
+                {
+                    _localization = GetClaimValue(claimType: ClaimConstants.Localization);
+
+                    if (_localization is null)
+                    {
+                        throw new UnauthorizedAccessException("Not authorized!");
+                    }
+                }
```
Comment (DmitriyFesak) [inline CharlieBackend.Business/Services/CurrentUserService.cs:103]: Can you do something like _localization = _localization ?? GetClaimValue(claimType: ClaimConstants.Localization) to remove one of these if statements please?

File: CharlieBackend.Panel/Services/CurrentUserService.cs [hunk 3]
```diff
@@ -114,7 +115,27 @@ public UserRole Role
...
+            get
+            {
+                if (_localization is null)
+                {
+                    _localization = GetClaimValue(ClaimsConstants.Localization);
+                    if (_localization is null)
+                    {
+                        throw new UnauthorizedAccessException("Not authorized!");
+                    }
+                }
```
Comment (DmitriyFesak) [inline CharlieBackend.Panel/Services/CurrentUserService.cs:130]: Same here

**Comment by UAKovalRoman** [comment]
https://github.com/ita-social-projects/WhatBackend/pull/932 - new pr, this one is deprecated

**Pull request merged**